README.md

```diff
@@ -1,2 +1,4 @@
 # ACS-UI-UX-capstone-project
-A project where we create a few webpages 
+A project where we create a few webpages. 
```

Comment:
Not sure why it removed a line, but shows underneath (showing commenting on multiple lines)

Reply:
comment recieved